src/cascadia/TerminalSettingsModel/TerminalSettings.h

```diff
@@ -166,6 +166,8 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         INHERITABLE_SETTING(Model::TerminalSettings, bool, RightClickContextMenu, false);
         INHERITABLE_SETTING(Model::TerminalSettings, bool, RepositionCursorWithMouse, false);
 
+        INHERITABLE_SETTING(Model::TerminalSettings, bool, ReloadEnvironmentVariables, true);

```

Comment:
qq: why does the control need to know about this?

Reply:
Control doesn't, but the page does. This is in the same weird case as `Elevate`. It's not a `Control` setting, but it is a per-control setting.